apps/conduit-e2e/src/integration/new-article.feature

```diff
@@ -0,0 +1,23 @@
+Feature: New article
```

Comment:
@stefanoslig I didn't include the negative path in this .feature file, e.g. Error message is displayed when any of the mandatory fields are missing.
I was thinking of creating a separate .feature file for that because of the Background steps in this one.
Another option is to include the negative scenarios in this file too, but move the Background steps inside of each scenario (which might be a pain to maintain later, due to copy-pasting).
What do you think?